app/utils/chart-config/api.ts

```diff
@@ -56,11 +56,7 @@ export const updateConfig = async (
           userId,
           data: {
             key,
-            version: state.version,
-            dataSource: state.dataSource,
-            layout: state.layout,
-            chartConfigs: state.chartConfigs,
-            activeChartKey: state.activeChartKey,
+            ...state,
```

Comment:
Not 100% sure, but I think it was destructured like this because there are more properties in the state that we want to keep in database? It looks like we don't want to have "state" property in database 👀